### PR TITLE
[Spanmetrics] - Add exemplars to events_total metric

### PR DESCRIPTION
**Pull request by aishyandapalli**
**Description:** Add exemplars to events_total metric for spanmetrics

### CODE REVIEW

**Review by songy23 — APPROVED** [review]
LGTM, pinging code owner @albertteoh for review.

**Comment by aishyandapalli** [comment]
@songy23 Thanks for a quick review. I forgot to add exemplars to events metric in my last PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/28671. So this is a minor enhancement to my last PR

**Review by albertteoh — APPROVED** [review]

File: connector/spanmetricsconnector/connector.go [hunk 1]
```diff
@@ -361,6 +361,9 @@ func (p *connectorImp) aggregateMetrics(traces ptrace.Traces) {
 							p.metricKeyToDimensions.Add(eKey, eAttributes)
 						}
 						e := events.GetOrCreate(eKey, eAttributes)
+						if p.config.Exemplars.Enabled && !span.TraceID().IsEmpty() {
```
Comment (codeboten) [inline connector/spanmetricsconnector/connector.go:364]: is there a test for this?

Comment (aishyandapalli) [reply connector/spanmetricsconnector/connector.go:364]: @codeboten Yes, the function in `connector_test.go` is **TestExemplarsForSumMetrics**

**Review by codeboten — APPROVED** [review]
thanks for addressing my question @aishyandapalli!

**Comment by aishyandapalli** [comment]
@codeboten Thanks for pointing it out :)

**Pull request merged**